client/dapp_chain_rpc_client.go

```diff
@@ -142,6 +142,18 @@ func (c *DAppChainRPCClient) Resolve(name string) (loom.Address, error) {
 	return loom.ParseAddress(addrStr)
 }
 
+func (c *DAppChainRPCClient) GetCode(contract string) ([]byte, error) {
```

Comment:
What is `contract` here? Can you document this please?

Reply:
Yes, it is the contract address, the point is this should be documented, should also document what the return value is.

Reply:
GetCode returns the runtime byte-code of a contract running on the DAppChain's EVM. Gives an error for non-EVM contracts.
contract - address of the contract in the form of a string. (Use loom.Address.String() to convert)
return []byte - runtime bytecode of the contract.